### PR TITLE
Make interface thunks forward variadic args

### PR DESCRIPTION
**Pull request by kinke**
This fixes issue #2613.

### CODE REVIEW

File: ir/irclass.cpp [hunk 1]
```diff
@@ -415,7 +415,8 @@ llvm::GlobalVariable *IrAggr::getInterfaceVtbl(BaseClass *b, bool new_instance,
       // call the real vtbl function.
       llvm::CallInst *call = gIR->ir->CreateCall(callee, args);
       call->setCallingConv(irFunc->getCallingConv());
-      call->setTailCallKind(llvm::CallInst::TCK_Tail);
+      call->setTailCallKind(thunk->isVarArg() ? llvm::CallInst::TCK_MustTail
+                                              : llvm::CallInst::TCK_Tail);
```
Comment (JohanEngelen) [inline ir/irclass.cpp]: is there a reason why this shouldn't always be a tail call?

Comment (kinke) [reply ir/irclass.cpp]: I got Phobos compile errors (std.regex) when always using must-tail, so I switched to this without further investigation.

Comment (JohanEngelen) [reply ir/irclass.cpp]: Hmm ok. LGTM otherwise. Perhaps we can investigate at some later moment

Comment (JohanEngelen) [reply ir/irclass.cpp]: Looks like `musttail` is causing problems on PowerPC: https://github.com/ldc-developers/ldc/issues/2686

**Pull request merged**